src/site/blocks/alert.drupal.liquid

```diff
@@ -0,0 +1,59 @@
+{% comment %}
+    Example data:
+    {
+        "entityBundle": "alert",
+        "fieldAlertType": "warning",
+        "fieldHelpText": null,
+        "fieldAlertTitle": "How do I talk to someone right now?",
+        "fieldReusability": "reusable",
+        "fieldAlertContent": {
+            "entity": {
+                "entityId": "932",
+                "entityBundle": "expandable_text"
+                "fieldWysiwyg": {
+                    "processed": "..."
+                },
+                "fieldTextExpander": "Find out how to get support anytime day or night."
+            }
+        }
+    }
+{% endcomment %}
+{% if alert.fieldAlertContent %}
+    {% assign expander = alert.fieldAlertContent.entity %}
+{% endif %}
+
+<div class="usa-alert usa-alert-{{ alert.fieldAlertType }}" role="alert">
+    <div class="usa-alert-body">
+        <h3 class="usa-alert-heading">
+            {{ alert.fieldAlertTitle }}
+            {% if expander.fieldTextExpander %}
+                <br><button id="{{ expander.entityBundle }}-{{ expander.entityId }}" class="va-button-link" aria-expanded="false">{{ expander.fieldTextExpander }}</button>
+            {% endif %}
+        </h3>
+        {% if alert.fieldHelpText %}<p>{{ alert.fieldHelpText }}</p>{% endif %}
```

Comment:
Will this exist for alerts with expandable content? Ideally the expanded content is directly below the button that expands it, without anything in between.

Reply:
good question that i wish i knew the answer to. currently, we can't even add that text to the system 😅 
i can move the button down to the bottom underneath that help text. hopefully, that will look fine.

Reply:
do these look alright? this is with putting the button markup underneath the `fieldHelpText`

Reply:
Yep, that looks ok to me.

Reply:
@jbalboni  pushed changes. thank you :)